src/components/ok-button/ok-button.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+import styles from './ok-button.module.scss'
+
+interface OkButtonProps {
+    text: string;
+}
+
+export const OkButton: FC<OkButtonProps> = ({text="ok"}) => (
```

Comment:
Well, not really agree with such name: `Ok` means you render some button with `Ok` text. Let's better name it like `AppButton` to make it more abstract and reusable.